docs/rfcs/002-archival_storage/README.md

```diff
@@ -514,46 +492,19 @@ Response:
   "@context": "https://api.wellcomecollection.org/bags/v1/context.json",
   "type": "Bag",
   "id": "born_digital/yy-yy-yy-yy",
-  "source": {
-    "type": "Source",
-    "id": "archivematica",
-    "label": "Archivematica"
-  },
   "space": {
     "id": "born_digital",
     "type": "Space"
   },
-  "identifiers": [
-    {
-      "type": "Identifier",
-      "identifierType": {
-        "id": "archivematica-aip-identifier",
-        "label": "Archivematica AIP identifier",
-        "type": "IdentifierType"
-      },
-      "value": "GC253_1046-a2870a2d-5111-403f-b092-45c569ef9476"
-    },
-    {
-      "type": "Identifier",
-      "identifierType": {
-        "id": "archivematica-aip-guid",
-        "label": "Archivematica AIP GUID",
-        "type": "IdentifierType"
-      },
-      "value": "a2870a2d-5111-403f-b092-45c569ef9476"
-    },
-    {
-      "type": "Identifier",
-      "identifierType": {
-        "id": "calm-alt-ref-no",
-        "label": "Calm Alt Ref No",
-        "type": "IdentifierType"
-      },
-      "value": "GC253/1046"
-    }
-  ],
+  "info": {
+    "type": "BagInfo",
+    "externalIdentifier": "a2870a2d-5111-403f-b092-45c569ef9476",
+    "externalDescription": "AIP title",
+    "payloadOxum": 435255.2,
```

Comment:
Be useful to know what the aspirational requirements around `payloadOxum` are? Do we want to verify this at some point (for MVP or after)?

Reply:
We should verify it before we verify checksums. It's intended as a fail fast.